homeassistant/components/media_player/universal.py

```diff
@@ -180,14 +181,20 @@ def _child_attr(self, attr_name):
     def _call_service(self, service_name, service_data=None,
                       allow_override=False):
         """Call either a specified or active child's service."""
+        if service_data is None:
+            service_data = {}
+
         if allow_override and service_name in self._cmds:
+            if service_data:
+                call_config = self._cmds[service_name]
+                call_config[ATTR_DATA].update(service_data)
```

Comment:
This is bad, you're changing the dictionary that is stored in `self._cmds`. Instead of updating the data, just pass the data as variables to `call_from_config`. This will allow people to use templates for their commands.